components/OAuth2.0/OauthHttpLocalServer.c

```diff
@@ -188,3 +187,29 @@ bool Spotify_StartMDNSService()
     ESP_LOGI(TAG, "MDNS Init done");
     return true;
 }
+
+/**
+ * @brief Run Http local service
+ */
+bool HttpServerServiceInit()
+{
```

Comment:
you should find a way to know the mdns `hostname` here to pass it the `Oauth_StartMDNSService` function

Reply:
done.